GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView.swift

```diff
@@ -99,6 +102,8 @@ extension CallVisualizer {
             layer.cornerRadius = props.style.cornerRadius
             renderProps()
             refreshButton.addTarget(self, action: #selector(refreshButtonTapped), for: .touchUpInside)
+            accessibilityElements = [titleLabel, closeButton, visitorCodeStack, poweredBy, refreshButton]
```

Comment:
Does VoiceOver say digit by digit or whole number?

Reply:
Digit by digit

Reply:
Would it be better to combine it into whole number? What do you think? @igorkravchenko @dukhovnyi @rasmustautsglia

Reply:
Apple recommends accessing individual elements within a stack view for accessibility with VoiceOver. This way, users can interact with each element separately, rather than having to navigate through the entire stack view as a single unit.

Reply:
@rasmustautsglia Makes sense.